tools/breaking-change-detector/GNUmakefile

```diff
@@ -29,22 +32,26 @@ endif
 	cd old/; \
```

Comment:
What happens if these folders don't exist? will the command still succeed?

Reply:
same as above